packages/console/src/pages/ApiResourceDetails/ApiResourcePermissions/index.tsx

```diff
@@ -8,14 +8,17 @@ import useSWR from 'swr';
 
 import ConfirmModal from '@/components/ConfirmModal';
 import PermissionsTable from '@/components/PermissionsTable';
+import { defaultPageSize } from '@/consts';
 import type { RequestError } from '@/hooks/use-api';
 import useApi from '@/hooks/use-api';
-import useTableSearchParams, { formatKeyword } from '@/hooks/use-table-search-params';
-import { buildUrl } from '@/utilities/url';
+import usePageSearchParameters from '@/hooks/use-page-search-parameters';
```

Comment:
usePaginateSearch?  usePageSearchParameters feels like a general useSearchParameters.

Reply:
renamed to `useSearchParameters`